documentation/publish.md

```diff
@@ -35,15 +35,15 @@ Learn how to get your extension signed and distributed worldwide or to your ente
 
 ## Get your extension signed
 
-Once your extension is coded and tested, it needs to meet the policies in the [developer agreement](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Agreement) and [add-on policies](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Reviews) before it is signed. If your extension doesn’t meet these policies, it may not get a signature or it could be [blocked](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Blocking_Process) after signing.
+Once your extension is coded and tested, it needs to meet the policies in the [developer agreement](/documentation/publish/firefox-add-on-distribution-agreement.md) and [add-on policies](/documentation/publish/add-on-policies.md) before it is signed. If your extension doesn’t meet these policies, it may not get a signature or it could be [blocked](/documentation/publish/add-ons-blocking-process.md) after signing.
```

Comment:
```suggestion
Once your extension is coded and tested, it needs to meet the policies in the [developer agreement](/documentation/publish/firefox-add-on-distribution-agreement) and [add-on policies](/documentation/publish/add-on-policies) before it is signed. If your extension doesn’t meet these policies, it may not get a signature or it could be [blocked](/documentation/publish/add-ons-blocking-process) after signing.
```